index.js

```diff
@@ -1,19 +1,44 @@
 var fs = require('fs');
-var tldjs = require('tldjs');
 
-var disposable = fs.readFileSync(__dirname + '/data/disposable.txt').toString().split('\n');
-var free = fs.readFileSync(__dirname + '/data/free.txt').toString().split('\n').concat(disposable);
+var disposable = load('/data/disposable.txt');
+var free = extend(load('/data/free.txt'), disposable);
 
 function isFree(email) {
     if (typeof email !== 'string') throw new TypeError('email must be a string');
-    var domain = tldjs.getDomain(email.split('@').pop());
```

Comment:
I believe tldjs is important here because it's how we handle email providers that sometimes offer wildcard subdomains. For example `foo.badmail.com` and `bar.badmail.com` might be represented once in the DB as `badmail.com`.

Is there a way to retain that behavior? I should probably add a test case for that.